monitor.cpp

```diff
@@ -13,17 +13,29 @@ void sleep() {
         sleep_for(seconds(1));
     }
 }
+
+bool checkThreshold(int lowThresh, int highThresh, float value) {
+    if ((value > highThresh) || (value < lowThresh)) {
+        return true;
+    }
+    return false;
+}
 int checkTemprature(int temperature) {
-    if (temperature > 102 || temperature < 95) {
-        cout << "Temperature is critical!\n";
+    if (checkThreshold(95, 102, temperature)) {
+        cout<< "Temperature is critical!\n";
         sleep();
         return 0;
     }
+   /* if (temperature > 102 || temperature < 95) {
```

Comment:
Don't leave commented code in commits. If you need it later, it will be there in the github history